astropy/wcs/wcsapi/low_level_api.py

```diff
@@ -252,7 +258,7 @@ def pixel_shape(self):
         return None
 
     @property
-    def pixel_bounds(self):
+    def pixel_bounds(self) -> Optional[Tuple[Tuple[float, float], ...]]:
```

Comment:
This disallows quantities in this `pixel_bounds` tuple (which `gwcs` was returning under some conditions) I assume we are all ok with this.

Reply:
I am!